src/vm/docs/mod.rs

```diff
@@ -324,7 +326,7 @@ associated with the expressions' paired key name.",
 };
 
 const TUPLE_GET_API: SpecialAPI = SpecialAPI {
-    input_type: "KeyName, Tuple|Void",
+    input_type: "KeyName, Tuple|Optional<Tuple>",
```

Comment:
It's unclear to me what's supposed to happen if `get` is passed an `Optional<Tuple>`

Reply:
It returns an Option type where the inner  type is the type of the asked-for field

Reply:
I feel like this should be documented.  Or, depending on how strict we want to be, we might want to consider having a variant of `get` that's just for `Tuple` and a distinct variant that's just for `Option<Tuple>`.

Reply:
Added to the docs output for `get`.

For now, I think it's fine for this to just be one function that accepts both `Tuple` and `Option<Tuple>` -- the typechecker will validate that you are using the function correctly.